src/lib/config.js

```diff
@@ -30,7 +30,7 @@ class Config {
     this.timezone = 0;
 
     this.locale = 'enUS';
-    this.os = 'Mac';
+    this.os = 'Win';
```

Comment:
https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Server/WorldSession.cpp

In function
void WorldSession::InitWarden

However it's a check done in various part of code and trinitycore uses Win
and OSX as identifiers.

So Win is quite universal and should be used.

I've tested and it works

Sorry i'm from mobile now i can t link you the exact line
Il 23/Mar/2016 15:02, "fallenoak" <notifications@github.com> ha scritto:

> In src/lib/config.js
> <https://github.com/wowserhq/wowser/pull/130#discussion_r57163394>:
>
> > @@ -30,7 +30,7 @@ class Config {
> >      this.timezone = 0;
> >
> >      this.locale = 'enUS';
> > -    this.os = 'Mac';
> > +    this.os = 'Win';
>
> Can you point to where this Warden check is implemented in either
> TrinityCore or cmangos?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wowserhq/wowser/pull/130/files/f887684ea16fa977cddedffdb0ae858ed117cd6b..19910a55512dae698949a65042a549946fcd41f2#r57163394>
>